examples/DatePicker.basic.jsx

```diff
@@ -3,16 +3,36 @@
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
 import * as React from 'react';
-import { DatePicker } from '@itwin/itwinui-react';
+import { DatePicker, IconButton, Popover } from '@itwin/itwinui-react';
+import { SvgCalendar } from '@itwin/itwinui-icons-react';
 
 export default () => {
   const [currentDate, setCurrentDate] = React.useState(new Date());
```

Comment:
Should we also display the selected date next to the button?

Reply:
We could, but then the Usage example will feel redundant, since the only thing currently differentiating it is the date display that it controls.

Reply:
The "Usage" section is done all wrong anyway. I think the point was to establish a difference between `DatePicker` used inside a `Popover` vs being embedded directly in a larger UI. It has nothing to do with selected dates.

Here's what I would suggest to improve the documentation:
1. Rename the "Usage" heading to "Standalone usage"
2. Change the `.usage.jsx` example underneath to show a DatePicker without `Popover` (and without `setFocus`).
3. Rename the top "Variants" heading to "Usage". Remove the "Basic" heading.
4. Add some text explaining the two different kinds of usages.
   - Show the `.basic.jsx` example directly underneath the text explaining `Popover` usage
   - Below this example, briefly explain "Standalone" usage and link to that heading which contains more details.

Reply:
Okay, will display dates for all examples and make the other suggested changes to the "Usage" section.

Reply:
Latest commit has the changes with regards to the "Usage" section of the documentation.